src/Foundation.Find.Cms/Config/EnumHelpers.cs

```diff
@@ -0,0 +1,58 @@
+﻿using EPiServer.Framework.Localization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done